frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md

```diff
@@ -972,6 +973,22 @@ Adds a new storage class provisioner as an option during storage class creation.
 
 ---
 
+## `console.tab`
+
+### Summary 
+
+Contributes a tab into an HorizontalNav component matching the sectionId.
+
+### Properties
+
+| Name | Value Type | Optional | Description |
+| ---- | ---------- | -------- | ----------- |
+| `contextId` | `string` | no | ContextId assigned to the horizontal nav in which the tab will be injected |
```

Comment:
```suggestion
| `contextId` | `string` | no | Context ID assigned to the horizontal nav in which the tab will be injected |
```

Reply:
Should we list the known context IDs?

Reply:
Inconsistencies in docs with `HorizontalNav` and `horizontal nav`

Reply:
Added a list of possible values. Decided for horizontal nav in the docs.